application/src/main/java/org/togetherjava/tjbot/commands/tags/TagManageCommand.java

```diff
@@ -280,6 +315,81 @@ private boolean isWrongTagStatusAndHandle(@NotNull TagStatus requiredTagStatus,
         return false;
     }
 
+    /**
+     * used to get a base {@link EmbedBuilder} for log embeds.
+     */
+    private EmbedBuilder getLogEmbed(@NotNull CommandInteraction event) {
+        return new EmbedBuilder()
+            .setAuthor(event.getUser().getAsTag(), null, event.getUser().getAvatarUrl())
+            .setColor(MOD_AUDIT_LOG_COLOR);
+    }
+
+    private void logCreateAction(@NotNull CommandInteraction event, @NotNull String id,
+            @NotNull String content) {
+        Guild guild = Objects.requireNonNull(event.getGuild());
+
+        switch (Subcommand.fromName(event.getSubcommandName())) {
+            case CREATE -> ModAuditLogWriter.log(guild,
+                    getLogEmbed(event).setTitle("Tag-Manage Create")
+                        .setDescription(String.format("created tag **%s**", id)),
+                    new VirtualFile(Filename.CONTENT.get(), content));
+
+            case CREATE_WITH_MESSAGE -> ModAuditLogWriter.log(guild,
+                    getLogEmbed(event).setTitle("Tag-Manage Create with message")
+                        .setDescription(String.format("created tag **%s**", id)),
+                    new VirtualFile(Filename.CONTENT.get(), content));
+
+            default -> throw new IllegalArgumentException("Subcommand Enum invalid");
```

Comment:
now its
``` java
default -> throw new IllegalArgumentException("Shouldn't log subcommand " + subcommand);
// in case the subcommand is "raw" or something else
```

Reply:
If you need a comment to make the error message clear, this indicates that the error message is not clear enough. Instead, remove the comment and improve the error message, like `"The subcommand '%s' is not intended to be logged to the mod audit channel".format(subcommand)`

Reply:
thank you, in fact, the error message you sent seems perfect for that, I'll change it to that

Reply:
done.